lib/acts_as_scrubbable/ar_class_processor.rb

```diff
@@ -0,0 +1,39 @@
+require 'acts_as_scrubbable/import_processor'
+require 'acts_as_scrubbable/update_processor'
+
+module ActsAsScrubbable
+  class ArClassProcessor
+    attr_reader :ar_class
+    private :ar_class
+
+    def initialize(ar_class)
+      @ar_class = ar_class
+
+      self.class_eval do
+        if ar_class.respond_to?(:import) && ActiveRecord::Base.connection_config[:adapter].include?("mysql")
+          include ImportProcessor
+        else
+          include UpdateProcessor
+        end
+      end
+    end
+
+    def process
+      # Removing any find or initialize callbacks from model
+        ar_class.reset_callbacks(:initialize)
+        ar_class.reset_callbacks(:find)
+
+        ActsAsScrubbable.logger.info "Scrubbing #{ar_class} ...".green
+
+        num_of_batches = Integer(ENV.fetch("SCRUB_BATCHES", "256")) if num_of_batches.nil?
```

Comment:
This variable is set here, so no need to check if it's already set.

```suggestion
        num_of_batches = Integer(ENV.fetch("SCRUB_BATCHES", "256"))
```